.github/workflows/validate-namespace.yaml

```diff
@@ -0,0 +1,68 @@
+name: Validate config-control Namespace
+
+on:
+  paths:
+    - 'dns-records/**/*.yaml'
+    - 'dns-records/**/*.yml'
+
+jobs:
+  validate-namespace:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Validate namespace
+      id: validate-namespace
+      run: |
+        files_with_namespace=()
+        yaml_files=()
+        EXPECTED_NAMESPACE='config-control'
+        
+        # Find yml/yaml files that have been changed    
+        changed_files=$(git diff --name-only HEAD^..HEAD)
+        echo "Changed Files"
+        echo "$changed_files"
+        
+        if [ -n "$changed_files" ]; then
+          while IFS= read -r file; do
+            if [[ "$file" =~ \.y(a)?ml$ ]]; then
+              yaml_files+=("$file")
+            fi
+          done < <(echo "$changed_files")
+
+          echo -e "\nFound yaml files:"
+          for file in "${yaml_files[@]}"; do
+            echo "$file"
+          done
+        else
+          echo -e "\nNo YAML files were changed."
+          exit 0  
+        fi
```

Comment:
Totally being a nitpick here, do you think it'd be helpful to collapse this into a single command -
```sh
yaml_files=($(git diff --name-only HEAD^..HEAD | grep -E '\.y(a)?ml$'))
```
The `grep -E ` would filter out `.yaml` / `.yml` files.


If this was meant to provide more verbose outputs to the user then please feel free to disregard this suggestion.